packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -188,6 +188,10 @@ const Popup: React.FC<PopupProps> &
         e.preventDefault();
         setPopupOpen(true, e);
       },
+      click: e => {
+        e.stopPropagation();
```

Comment:
If there is a reason to `stopPropagation()` let's add a comment why

Reply:
Not really, removed!